caiman/source_extraction/cnmf/cnmf.py

```diff
@@ -324,8 +324,13 @@ def fit_file(self, motion_correct=False, indices=None, include_eval=False):
                 perform analysis only on a part of the FOV
             include_eval (bool)
                 flag for performing component evaluation
+            output_dir (str)
+                directory to save the outputs
+            return_mc (bool)
+                flag to return motion correction object
```

Comment:
We should saveguard against users setting `return_mc=True` but `motion_correct=False`, conflicting input values.

Perhaps with something like this, somehwere
```
return_mc = return_mc if motion_correct else False
```

Reply:
Line 400 fixes this issue